OTGTool/src/main/scala/t/sparql/ListManager.scala

```diff
@@ -161,19 +161,31 @@ abstract class ListManager[T](config: TriplestoreConfig) extends Closeable {
       s"WHERE { ?s ?p <$defaultPrefix/$name> . } ")
   }
 
+  //Update a predicate of an item that is supposed to have at most one value
   def updateSinglePredicate(name: String, key: String, value: String): Unit = {
-       triplestore.update(s"$tPrefixes\n " +
-        s"DELETE { <$defaultPrefix/$name> $key ?o. } \n" +
-        s"WHERE { <$defaultPrefix/$name> $key ?o. } ")
     triplestore.update(s"$tPrefixes\n " +
-        s"INSERT DATA { <$defaultPrefix/$name> $key $value }")
+      s"DELETE { <$defaultPrefix/$name> $key ?o. } \n" +
+      s"WHERE { <$defaultPrefix/$name> $key ?o. } ")
+    triplestore.update(s"$tPrefixes\n " +
+      s"INSERT DATA { <$defaultPrefix/$name> $key $value }")
+  }
+
+  def setTimestamp(name: String): Unit = {
+    val encodedDate = dateFormat.format(new Date())
+    updateSinglePredicate(name, timestampRel, "\"" + encodedDate + "\"")
   }
 
   def setComment(name: String, comment: String): Unit = {
-    updateSinglePredicate(name, "t:comment", "\"" + comment + "\"")
+    if (comment.contains('"')) {
```

Comment:
This contains(") checking should have been here all along, so I think it was an opening for bugs or weird data issues. I decided to fix it while I was working here.